src/frontend/komponenter/Behandling/Personopplysninger/Sivilstatus.tsx

```diff
@@ -29,6 +37,7 @@ const Sivilstatus: React.FC<{ sivilstander: ISivilstand[] }> = ({ sivilstander }
                                         <Normaltekst>-</Normaltekst>
                                     )}
                                 </td>
+                                {harDødsdato && <td>{sivilstand.dødsdato}</td>}
```

Comment:
Trenger vi både etikett og dødsdato? Eller holder det med dødsdatoen?

Reply:
Ja, eller vi har en liten diskusjon om det :P 
https://nav-it.slack.com/archives/C01087QRJ2U/p1624965749196500